101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md

```diff
@@ -19,30 +19,27 @@
 
 # Concepts
 
-## Web Actions
+## Web actions
 
-Cloud Function actions can be annotated with a special flag (i.e., `--web true`) at creation to convert them into "web actions". The result is the corresponding creation of a public URL that can be used to trigger the action from any web app.
+Cloud Function actions can be annotated with a special flag, such as`--web true`, at creation to convert them into [web actions](https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-actions_web). The result is the corresponding creation of a public URL that can be used to trigger the action from any web app.
 
-Web actions can then also be invoked via HTTP requests without user authentication where the HTTP request parameters are automatically converted in event parameters. Web actions are able to control the HTTP response headers and body to support any content-types, manage cookies, perform HTTP redirects directly.
+Web actions can then be invoked via HTTP requests without user authentication where the HTTP request parameters are automatically converted in event parameters. Web actions are able to control the HTTP response headers and body to support any content types, manage cookies, and perform HTTP redirects directly.
 
-In addition, requesters can specify the `content-type` they want for the response as one of `.json`, `.html`, `.http`, `.svg` or `.text`. For convenience, the `.http` extension is the default.
```

Comment:
There is a limited (fixed set) of supported content types so it is "one of" that set

Reply:
It just isn't grammatically correct. Can it be "as one of the following: `.json`..."?

Reply:
sure